### PR TITLE
Signuppage ui fix

### LINKED ISSUE #140: Sign Up screen is clipped on phone
**Describe the bug**
The sign up screen is horizontally clipped on an iPhone7 due to hardcoded min widths. 

**To Reproduce**
1. Go to staging.hknucsd.com/signup

**Expected behavior**
Expected whole screen to be visible

**Smartphone (please complete the following information):**
 - Device: iPhone 7
 - OS: Doesn't matter here
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by godwinpang**
closes #140 

kinda wip but you can preview it now - I'll go ahead and fix the minwidth stuff on the dropdown as well

edit: completely done :)

### CODE REVIEW

File: src/pages/SignUpPage/components/SignUpForm/styles.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const styles = () => ({
-  signUpButton: {
-    marginTop: '10px',
+  signUp: {
+    marginTop: 8,
```
Comment (thaigillespie) [inline src/pages/SignUpPage/components/SignUpForm/styles.js:3]: What does 8 mean here? 8 pixels?

File: src/pages/SignUpPage/styles.js [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
-  },
-  signUpCard: {
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
-    width: '536px',
-    marginTop: '37px',
-    marginBottom: '37px',
-    paddingBottom: '20px',
+    [theme.breakpoints.up('sm')]: {
```
Comment (thaigillespie) [inline src/pages/SignUpPage/styles.js:3]: What is this? Just curious cuz I really don't know LOL

File: src/pages/SignUpPage/components/SignUpForm/styles.js (hunk 1 above)
Comment (godwinpang) [reply src/pages/SignUpPage/components/SignUpForm/styles.js:3]: yeah 8 converts to px for material ui css :)

File: src/pages/SignUpPage/styles.js (hunk 2 above)
Comment (godwinpang) [reply src/pages/SignUpPage/styles.js:3]: https://material-ui.com/customization/breakpoints/

Comment (godwinpang) [reply src/pages/SignUpPage/styles.js:3]: just a nice wrapper around media queries that mui gives us

**Pull request merged**